server/sonar-server/src/test/java/org/sonar/server/project/ws/BulkDeleteActionTest.java

```diff
@@ -36,98 +41,158 @@
 import org.sonar.server.organization.BillingValidationsProxy;
 import org.sonar.server.organization.TestDefaultOrganizationProvider;
 import org.sonar.server.tester.UserSessionRule;
-import org.sonar.server.ws.WsTester;
+import org.sonar.server.ws.TestResponse;
+import org.sonar.server.ws.WsActionTester;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.Matchers.any;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.verifyZeroInteractions;
+import static org.sonar.api.utils.DateUtils.formatDate;
+import static org.sonar.db.component.SnapshotTesting.newAnalysis;
 import static org.sonar.db.permission.OrganizationPermission.ADMINISTER;
+import static org.sonarqube.ws.client.project.ProjectsWsParameters.PARAM_ANALYZED_BEFORE;
+import static org.sonarqube.ws.client.project.ProjectsWsParameters.PARAM_ON_PROVISIONED_ONLY;
+import static org.sonarqube.ws.client.project.ProjectsWsParameters.PARAM_ORGANIZATION;
+import static org.sonarqube.ws.client.project.ProjectsWsParameters.PARAM_PROJECTS;
+import static org.sonarqube.ws.client.project.ProjectsWsParameters.PARAM_QUALIFIERS;
 
 public class BulkDeleteActionTest {
```

Comment:
Correct me if I'm wrong, but it seems to me that the main use case of the ticket is not covered : checking that more than 50 projects can be deleted at once.
I see it in the IT but I think it would be great to have a UT